### PR TITLE
Vessels mask UI

### PR DESCRIPTION
**Pull request by Baschtien**
WARNING: This is not a merge on dev!

- Formatted the UI for the analysis panel.
- Added settings as shown in the prototype of the panel both in the UI and in settings.hh
- Includes all changes from Otsu as well (the branch is add_ostsu)

(Delete branch after merge)

### CODE REVIEW

File: Holovibes/sources/gui/windows/MainWindow.cc [hunk 1]
```diff
@@ -317,7 +318,11 @@ void MainWindow::on_notify()
     }
 
     ui_->CompositePanel->setHidden(api::get_compute_mode() == Computation::Raw ||
-                                   (api::get_img_type() != ImgType::Composite));
+                                   api::get_img_type() != ImgType::Composite);
+
+    const int img_type_int = static_cast<int>(api::get_img_type());
+    ui_->AnalysisPanel->setEnabled(img_type_int >= static_cast<int>(ImgType::Moments_0) &&
+                                   img_type_int <= static_cast<int>(ImgType::Moments_2));
```
Comment (Hpn4) [inline Holovibes/sources/gui/windows/MainWindow.cc:325]: Move these lines before the condition with `api::get_is_computation_stopped()` and replace `setEnabled` by `setVisible`

**Review by ArthurCourselle — DISMISSED** [review]
Change otsu's functions name from camelcase to snakecase for consistency

File: Holovibes/sources/gui/windows/MainWindow.cc (hunk 1 above)
Comment (Baschtien) [reply Holovibes/sources/gui/windows/MainWindow.cc]: Fixed

**Review by Hpn4 — APPROVED** [review]

File: Holovibes/includes/gui/windows/panels/view_panel.hh [hunk 2]
```diff
@@ -65,6 +65,11 @@ class ViewPanel : public Panel
      *
      */
     void set_artery_mask(bool value);
+    /*!
+     * \brief Enables or Disables Otsu
+     *
+     */
+    void set_otsu(bool value);
```
Comment (ArthurCourselle) [inline Holovibes/includes/gui/windows/panels/view_panel.hh:72]: From otsu i think it should be in analysis panel

**Review by ArthurCourselle — APPROVED** [review]

**Pull request merged**